spec/admin/target_spec.rb

```diff
@@ -8,16 +8,18 @@
   end
 
   describe 'Show all targets' do
-    it 'displays all targets for the admin user' do
```

Comment:
I think we want to keep this spec to "displays all targets" and add the new scenario where you can filter by params

Reply:
Yep you are wright. Thanks!